As pointed out by Paul forgey in bug #96818, makefile was defining as constant
overwriting the passed values for the three _ENABLED variables.
This patch makes them take a default value if not set otherwise, allowing us to
pass different values during make stage.

Index: daapd-0.2.4a/makefile
===================================================================
--- daapd-0.2.4a.orig/makefile
+++ daapd-0.2.4a/makefile
@@ -1,8 +1,8 @@
 # configure daapd here
 
-HOWL_ENABLE = 0
-ZLIB_ENABLE = 1
-MPEG4_ENABLE = 0
+HOWL_ENABLE := 0
+ZLIB_ENABLE := 1
+MPEG4_ENABLE := 0
 
 
 # no need to touch anything below this line
